source/blog/2021-03-07-this-month-in-solidus-february-2021.html.erb.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: "This Month in Solidus: February 2021"
+date: 2021-03-07
+tags: Solidus
+author: Alessandro Desantis
+cover_image: /blog/shared/images/solidus-monthly-update.png
+description: >
+  Dive in and see everything that happened in the Solidus world in February 2021!
+---
+
+Hey there, and welcome to your usual This Month in Solidus! Let's dive in and see everything that
+happened in the Solidus world this February.
+
+### New Full-Time Maintainer 👷‍♂️
+
+Yes, you read that right! So far, Solidus has progressed thanks to the generous contributions of its
+community, but we felt it was time we gave it a proper full-time maintainer, and
+[Nebulab](https://nebulab.it) stepped in to fill the gap.
+
+[Marc Busqué](https://github.com/waiting-for-dev) is a senior Ruby developer based in beautiful
+[Cambrils, Spain](https://en.wikipedia.org/wiki/Cambrils). He has an impressive track record of
+contributing to OSS, having worked on the renowned [dry-rb](https://github.com/dry-rb) ecosystem.
+
+Marc recently joined the Nebulab team, where he will work with the Core Team on the maintenance and
+evolution  of Solidus, splitting his time between keeping the platform in good shape and pushing it
+forward with his technical expertise.
+
+Welcome aboard Marc, and best of luck!
+
+P.S. Pretty soon, Marc should be in good company... Stay tuned!
+
+### Backend UX Audit 🎨
+
+Last Thursday, the stakeholders have approved a vote to engage [Baymard](https://baymard.com/) for
+a UX audit of the Solidus backend.
+
+We are _extremely_ excited about this: Baymard is a world-class UX institute that has worked with
+some of the best players in the industry, and we're looking forward to using their expertise to
+improve the Solidus backend experience.
+
+Baymard can use all the help they can get, so if you have any feedback to share about the Solidus
+backend that you think would be beneficial for this audit, don't be shy about stepping forward! You
+can reach out to any Core Team member on Slack, who will direct you to the proper channels.
```

Comment:
Should we emphasize that this will be paid with the money we collected together?

Reply:
@kennyadsl done!